static/src/stylesheets/module/journalism/_media.scss

```diff
@@ -302,9 +302,19 @@
     border-radius: 30px;
     height: 34px;
 
-    .from-content-api.from-content-api {
+    .from-content-api.from-content-api & {
```

Comment:
I haven't seen this convention to add `&` at the end of a parent selector. Normally I see it at the start of a child selector (is it just a different way of writing this?